client/components/aboutUs/Resources.js

```diff
@@ -0,0 +1,310 @@
+        fill="#18E28C"
+      />
+    </svg>
+  );
+};
+
+const repoLink = (
+  <a
+    href="https://github.com/chingu-voyages/moonshot-chingu-quiz"
+    rel="noreferrer noopener"
+    target="_blank"
+  >
+    repo
+  </a>
+);
+
+const repoGitLink = (
+  <a
+    href="https://github.com/chingu-voyages/moonshot-chingu-quiz"
+    rel="noreferrer noopener"
+    target="_blank"
+  >
+    https://github.com/chingu-voyages/moonshot-chingu-quiz
+  </a>
+);
+
+const contributeLink = (
+  <Link href="/contribute">
+    <a>page on contributing</a>
+  </Link>
+);
+
+const chinguLink = (
+  <a href="https://Chingu.io" rel="noreferrer noopener" target="_blank">
+    Chingu
+  </a>
+);
+
+const chinguIOLink = (
+  <a href="https://Chingu.io" rel="noreferrer noopener" target="_blank">
+    Chingu.io
+  </a>
+);
+
+const issueLink = (
+  <a
+    href="https://github.com/chingu-voyages/moonshot-chingu-quiz/issues"
+    rel="noreferrer noopener"
+    target="_blank"
+  >
+    here
+  </a>
+);
+
+export const HEADER_DATA = {
+  id: 0,
+  title: "How did this project start?",
+  text: (
+    <>
+      <p>
+        The idea started with a group of ({chinguLink}) developers who saw a
+        need in the community for a tool to help people prepare for technical
+        interviews. <br />
+      </p>
+      <br />
+      <p>
+        This app aims to help fellow developers get a better sense of weak
+        points in their knowledge base, provide resources to improve in those
+        areas, and offer some ways to better prepare for technical interviews
+      </p>
+    </>
+  ),
+};
+
+export const ABOUT_DATA = [
+  {
+    id: 1,
+    svg: <MessageSVG />,
+    title: "What in the world is Chingu?",
+    text: (
+      <p>
+        Chingu is a developer community that strives to help people level-up by
+        building real projects in a team setting with others focused on
+        improving their skill set. The benefits extend past tech and into
+        getting a better grip on the soft-skills needed when working in a team
+        environment; things like working through project ideas, resolving merge
+        conflicts, and planning sprints. Learn more at ({chinguIOLink})
```

Comment:
The parenthesis around the links is not needed: `Chingu.io` instead of `(Chingu.io)`. 
When I wrote the text I added them to signify to the designer they were links and they got left in 😅 . They can be removed from around any links, and the text in them should just be hyperlinked.